### PR TITLE
[BE-15] constraints were changed in liquibase scripts

### PR DESCRIPTION
**Pull request by shpaser2**

### CODE REVIEW

**Review by AlKaliada — DISMISSED** [review]

**Review by nazfurdychka — DISMISSED** [review]
please change entities column annotations so that hibernate mapping will correspond liquibase changing scripts

**Review by nazfurdychka — APPROVED** [review]

**Review by AlKaliada — DISMISSED** [review]

**Review by nazfurdychka — DISMISSED** [review]

File: src/main/java/com/exadel/project/internship/entity/Internship.java [hunk 1]
```diff
@@ -54,6 +54,7 @@
 
     @Enumerated(EnumType.STRING)
     @Column(name = "format", columnDefinition="ENUM('ONLINE','OFFLINE')")
+//    @Column(name = "format")
```
Comment (serkushner) [inline src/main/java/com/exadel/project/internship/entity/Internship.java:57]: Pls remove redurant comment

File: src/main/resources/db/changelog/changelog-constraints-added.yaml [hunk 2]
```diff
@@ -0,0 +1,255 @@
+databaseChangeLog:
+  - changeSet:
+      id: 20210331.[BE-15]-simplify_primary_key
+      author: shpaser3@yandex
+      comment: "PK changed from 3 columns based to one column based"
+      preConditions:
+        - onFail: MARK_RAN
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-constraints-added.yaml:7]: Empty precondition?

**Review by serkushner — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changelog-constraints-added.yaml [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+            tableName: additional_info
+        - dropColumn:
+            columnName: oldId
+            tableName: additional_info
+
+  - changeSet:
+      id: 20210331.[BE-15]-additional_info-unique
+      author: shpaser3@yandex
+      preConditions:
+        - onFail: MARK_RAN
```
Comment (serkushner) [inline src/main/resources/db/changelog/changelog-constraints-added.yaml:41]: The same

**Review by serkushner — CHANGES_REQUESTED** [review]

**Review by serkushner — DISMISSED** [review]

**Review by AlKaliada — APPROVED** [review]

**Review by nazfurdychka — APPROVED** [review]

**Pull request merged**